core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.html

```diff
@@ -7,6 +7,10 @@
            class="progress">
         <div class="progress-bar bg-success"
              role="progressbar"
+             [attr.aria-label]="progressBarLabel"
+             title="progressBar"
+             container="body"
+             placement="bottom"
```

Comment:
Sorry, but are we sure these attributes are necessary?

Reply:
Yes it was added because chromevox reads the title name .
Hence  I added the same .

Reply:
Okay, but what about `container` and `placement`?